homeassistant/components/intellifire/config_flow.py

```diff
@@ -220,10 +221,12 @@ async def async_step_user(
         LOGGER.debug("Running Step: manual_device_entry")
         return await self.async_step_manual_device_entry()
 
-    async def async_step_reauth(self, user_input=None):
+    async def async_step_reauth(self, entry_data: Mapping[str, Any]) -> FlowResult:
         """Perform reauth upon an API authentication error."""
         LOGGER.debug("STEP: reauth")
         entry = self.hass.config_entries.async_get_entry(self.context["entry_id"])
+        assert entry
+        assert entry.unique_id
```

Comment:
Not sure why these asserts are added.  What action should we take if they fail?

Reply:
They are added to satisfy mypy.
If either was None, then it would fail anyway further down.